src/ai/susi/server/api/susi/SusiService.java

```diff
@@ -90,6 +90,10 @@ public ServiceResponse serviceImpl(Query post, HttpServletResponse response, Aut
         String skill_name = post.get("skill", "");
         Boolean include_default_skills = true;
 
+        if (post.get("excludeDefaultSkills", null) != null) {
```

Comment:
This is extremely confusing. What does that mean? If I pass "yes", it has the opposite meaning. Why would an option named "excludeDefaultSkills" should set a variable "excludeDefaultSkills"? With opposite meaning?

Please use one name for options and variables. Explain what the default is. Pass a boolean value to change the meaning. Make the default of the boolean value "false", because that would be natural.